src/index.html

```diff
@@ -15,8 +15,51 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img src="./images/logo.png"
+      alt="Logo"
+      class="img_logo">
+      <nav>
+        <ul class="nav_list">
+          <li class="nav_line">
+            <a href="#" class="nav_link active">Apple</a>
+            <div class="is-active"></div>
+          </li>
+          <li class="nav_line">
+            <a href="#" class="nav_link">Samsung</a>
+            <div class="is-active"></div>
+          </li>
+          <li class="nav_line">
+            <a href="#" class="nav_link">Smartphones</a>
+            <div class="is-active"></div>
+          </li>
+          <li class="nav_line">
+            <a href="#" class="nav_link">Laptops & Computers</a>
+            <div class="is-active"></div>
+          </li>
+          <li class="nav_line">
+            <a href="#" class="nav_link">Gadgets</a>
+            <div class="is-active"></div>
+          </li>
+          <li class="nav_line">
+            <a href="#" class="nav_link">Tablets</a>
+            <div class="is-active"></div>
+          </li>
+          <li class="nav_line">
+            <a href="#" class="nav_link">Photo</a>
+            <div class="is-active"></div>
+          </li>
+          <li class="nav_line">
+            <a href="#" class="nav_link">Video</a>
+            <div class="is-active"></div>
+          </li>
+        </ul>
+      </nav>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to improve readability. There should be an empty line between the closing `</li>` and the next `<li>`.